rpgJavaInterpreter-core/src/main/kotlin/com/smeup/rpgparser/interpreter/internal_interpreter.kt

```diff
@@ -694,10 +694,10 @@ class InternalInterpreter(val systemInterface: SystemInterface) {
         }
     }
 
-    private fun toSearchValues(searchArgExpression: Expression): List<Field> {
+    private fun toSearchValues(searchArgExpression: Expression): List<RecordField> {
```

Comment:
Perhaps we could move methods specific to database into an helper class, with the goal of simplifying the interpreter class